src/bpftrace.cpp

```diff
@@ -52,6 +52,7 @@ namespace bpftrace {
 DebugLevel bt_debug = DebugLevel::kNone;
 bool bt_quiet = false;
 bool bt_verbose = false;
+bool bt_verbose2 = false;
```

Comment:
Nit: just make `bt_verbose` an integer?

Reply:
I like adding bt_verbose2 because it's simple ;-) there are many bt_verbose condition instances of that would need to change.. maybe I'm biased from perf code but that seems more clear to me than adding other machinery/funcs to print messages with proper verbose level